legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-core/src/main/java/org/finos/legend/engine/persistence/components/relational/api/RelationalIngestorAbstract.java

```diff
@@ -492,6 +532,18 @@ private List<IngestorResult> ingest(List<DataSplitRange> dataSplitRanges, Schema
         }
     }
 
+    private List<DataError> performDryRun()
+    {
+        if (enrichedIngestMode instanceof BulkLoad)
+        {
+            return relationalSink().performDryRun(executor, generatorResult.dryRunSqlPlan(), sampleRowCount());
```

Comment:
Need to execute dry run preactions - I remember we discussed about this but maybe the code hasnt been pushed?

Reply:
Yes, will push the code while addressing other comments